### PR TITLE
htmlcss completed

### PR DESCRIPTION
**Pull request by atmakurramakanth**

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About Me</a></li>
+                <li><a href="contact.html">Contact Me</a></li>
+                <li><a href="gallery.html">My Gallery</a></li>
+            </ul>
+            <a class="header-call" href="tel:9390362131">Call</a>
+        </nav>
+    </header>
```
Comment (AbhilashZemoso) [inline contact.html]: Remove duplicate code

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <li><a href="gallery.html">Gallery</a></li>
+        </ul>
+        <ul class="footer-links-add">
+            <li><a href="#"> Address </a></li>
+            <li><p>ghatkesar near Sreenidhi college</p></li>
+            <li><p>Hyderabad-509301</p></li>
+            <li><p>Telangana</p></li>
+        </ul>
+   
+    </footer>
```
Comment (AbhilashZemoso) [inline contact.html]: remove duplicate code

**Review by atmakurramakanth — COMMENTED** [review]
duplication removd

**Review by AbhilashZemoso — APPROVED** [review]
LGTM

**Pull request merged**